web/components/Users/Results.tsx

```diff
@@ -115,6 +115,9 @@ export default function Results({ filters, setSearched }: ResultsProps) {
         )}
         <Divider />
       </Stack>
+      {(!users || !users.data || users.data.length == 0) && (
+        <Text align={"center"}>No results found.</Text>
+      )}
```

Comment:
This ternary should be modified so that we render the below `Flex` block only when there is data